charts/chronicle/templates/chronicle-init.yaml

```diff
@@ -159,13 +159,17 @@ spec:
           command: [ "bash", "-ec"]
           args:
             - |
-              if opactl \
-                  --sawtooth-address tcp://$HOST:$PORT \
-                    get-policy \
-                      --id {{ .Values.opa.policy.id }} \
-                      --output policy.bin >/dev/null 2>&1; then
+              echo "Attempting to get policy."
+              opactl \
+                --sawtooth-address tcp://$HOST:$PORT \
+                  get-policy \
+                    --id {{ .Values.opa.policy.id }} \
+                    --output policy.bin
+
+              if [ -f "policy.bin" ]; then
                 echo "Policy already set."
                 touch /shared-data/policy-already-set
+                exit 0
```

Comment:
Does this `exit 0` or the following after the `echo` do anything useful? Are we trying to ignore errors from the `touch`?

Reply:
It immediately signals to a reader that this is a successful exit.